### ISSUE TITLE
Correcting a name seems to have no effect

### ISSUE THREAD

**Issue by tmtmtmtm** [mysociety/verification-pages#177]
When verifying the data for Muttamsetti Srinivasa Rao in the [16th Lok Sabha](https://www.wikidata.org/wiki/User:Verification_pages_bot/verification/in/Member_of_the_16th_Lok_Sabha), I said that the source lists him as **Mutham**setti Srinivasa Rao. This updated how he displays on the Verification page:

![screen shot 2018-06-28 at 11 44 16](https://user-images.githubusercontent.com/57483/42029816-b170347c-7ac8-11e8-8779-f72156d25d1b.png)

but seems to have had no effect on his Wikidata item:

![screen shot 2018-06-28 at 11 45 29](https://user-images.githubusercontent.com/57483/42029845-c8547acc-7ac8-11e8-89d6-684bd84939de.png)

https://www.wikidata.org/w/index.php?title=Q16901183&type=revision&diff=703692331&oldid=657375531

**Comment by tmtmtmtm** [comment]
I think this is no longer relevant after #355
